Lombiq.ContentEditors/Views/AsyncEditor.Component.Editor.cshtml

```diff
@@ -0,0 +1,41 @@
+<style>
+    .cursor-pointer {

```

Comment:
You can use [`role="button"`](https://stackoverflow.com/questions/47269189/css-class-for-pointer-cursor) instead which better reflects their purpose anyway.

Reply:
Do you need to use old Bootstrap (pre June 2020)? If not, then you don't need the style element at all because `[role="button"]` is given by Bootstrap (see link in previous comment). For example in OSOCE you have this role styling out of the box.